python/examples/integrations/fastapi/README.md

```diff
@@ -0,0 +1,139 @@
+
+Finally, we'll create an prediction API endpoint with FastAPI that will also log the input + prediction data with whylogs and write the data profile to WhyLabs.
+
+```python
+
+# Create Pydantic model for input data
+class PredictRequest(BaseModel):
+    sepal_length: float
+    sepal_width: float
+    petal_length: float
+    petal_width: float
+
+# Create prediction function
+def make_prediction(model, features):
+
+    results = model.predict(features)
+    probs =  model.predict_proba(features)
+    result = results[0]
+
+    output_cls = CLASS_NAMES[result]
+    output_proba = max(probs[0])
+
+    return (output_cls, output_proba)
+
+# Create API endpoint
+@app.post("/predict")
+def predict(request: PredictRequest) -> JSONResponse:
+
+    # Convert input data to list for model prediction
+    data = jsonable_encoder(request)
+    features = list(data.values())
+ 
+    # Make prediction with model & add to data dictionary
+    predictions = make_prediction(model, [features])
+    data['model_class_output'] = predictions[0]
+    data["model_prob_output"] = predictions[1]
+    
+    # Log input data with whylogs & write to WhyLabs
+    profile_results = why.log(data)
+    profile_results.writer("whylabs").write()
+    
+    return JSONResponse(data)
+```
+
+You can now make a prediction request to the API endpoint. The input data will be logged with whylogs and written to WhyLabs for ML monitoring.
+
+## 🚀 How to run the example:
+
+### 1. Install dependencies
+
+```bash
+pip install -r requirements.txt
+```
+
+### 2. Train the model
+
+This will train and save an example model as `knn_model.pkl`
+```bash
+python train.py
+```
+
+### 3. Run the API server
+First update the WhyLabs variables in `api.py` with your own values. Then run the API server.
+```bash
+uvicorn api:app --reload
+```
+
+### 4. Make a prediction request
+You can use the following cURL command to make a prediction request to the API endpoint or use the swagger UI at `http://localhost:8000/docs`.
+
+```bash
+curl -X POST "http://localhost:8000/predict" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"sepal_length\": 5.1, \"sepal_width\": 3.5, \"petal_length\": 1.4, \"petal_width\": 0.2}"
+```
+
+### 5. View the logged data in WhyLabs
+Go to the [WhyLabs dashboard](https://app.whylabsapp.com/) and click on the project setup for logging. 
+
+Once profiles are written to WhyLabs they can be inspected, compared, and monitored for data quality and data drift.
+
+![detecting data drift with WhyLabs](https://whylabs.ai/_next/image?url=https%3A%2F%2Fcontent.whylabs.ai%2Fcontent%2Fimages%2F2022%2F11%2FFrame-6899--1-.png&w=3120&q=75)
+
+Learn more about how to use WhyLabs to monitor your ML models in production [here](https://docs.whylabsapp.com/).
+
+## A decoupled architecture approach to ML monitoring
```

Comment:
this was rephrased by Anthony with a more neutral language. Might copy from it https://docs.whylabs.ai/docs/fastapi-integration#decoupling-whylogs-from-fastapi